packages/esm-form-entry-app/src/app/form-creation/form-creation.service.ts

```diff
@@ -111,6 +113,10 @@ export class FormCreationService {
     this.dataSources.registerDataSource('patient', this.formDataSourceService.getPatientObject(patient), true);
     this.dataSources.registerDataSource('rawPrevEnc', createFormParams.previousEncounter, false);
     this.dataSources.registerDataSource('userLocation', createFormParams.user.sessionLocation);
+    this.dataSources.registerDataSource(
+      'mostRecentObsValue',
+      this.formDataSourceService.getMostRecentObsValueBefore(date, patientUuid),
```

Comment:
Also, note that the patient UUID should already be available inside the form... The second parameter here was supposed to be a list of concepts.